app/code/community/Reverb/ReverbSync/Helper/Orders/Creation/Address.php

```diff
@@ -22,8 +26,10 @@ public function addOrderAddressAsShippingAndBillingToQuote(stdClass $reverbOrder
         $shippingAddressObject = $reverbOrderObject->shipping_address;
         if (!is_object($shippingAddressObject))
         {
-            $error_message = $this->__(self::ERROR_NO_ADDRESS);
-            throw new Exception($error_message);
+            // In this event, we still want to create the Magento order. We will populate the address fields with a
+            //      fake placeholder value
+            $this->_addNoAddressToOrderObject($reverbOrderObject);
+            $shippingAddressObject = $reverbOrderObject->shipping_address;
```

Comment:
what does magento require in terms of a "full address"? What if we sent an address without a country code for example? We shouldn't do that in general but I'm wondering if we need to ensure that every magento side address field is filled even if it's missing from reverb